cfpb/consumerfinance.gov
# Ask CFPB: Move Tip to Ask

The only non-ask page I can find via the block inventory that uses this "Tip" inset is this https://www.consumerfinance.gov/about-us/blog/prestamos-estudiantiles-y-coronavirus/

This PR moves the Tip to the ask templates and makes a dedicated Content block template for the Ask pages, to differentiate it from the content block used across page types. 

The tip inset on https://www.consumerfinance.gov/about-us/blog/prestamos-estudiantiles-y-coronavirus/ will need to be refactored into something else. Either a text block or a full-width-text related links block without links. 

## Changes

- Ask CFPB: Move Tip to Ask


## How to test this PR

1. `python cfgov/manage.py migrate` and search the blog inventory for "Tip" and see that there are only ask pages that have Tip insets.
